manim/mobject/svg/brace.py

```diff
@@ -70,14 +72,14 @@ def construct(self):
     def __init__(
         self,
         mobject: Mobject,
-        direction: Vector3D | None = DOWN,
+        direction: Vector3D = DOWN,
```

Comment:
Might this be typed as `Point3DLike` in the meantime?
```suggestion
        direction: Point3DLike = DOWN,
```